code/modules/admin/watchlist.dm

```diff
@@ -36,6 +36,8 @@
 	if(browse)
 		watchlist_show(target_sql_ckey)
 
+	add_note(target_ckey, "Added to Watchlist - [reason]", null, usr.ckey, 0, null, 0)
```

Comment:
that i can tell you

@ExcessiveUseOfCobblestone why the fuck would I tell someone i watchlisted them they'll just point me out in appeals